src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -104,7 +104,7 @@ public Admission getAdmission(int id) throws OHServiceException {
 	 * @param patient the patient target of the admission.
 	 * @return the patient admission or {@code null} if the operation fails.
 	 */
-	public Admission getCurrentAdmission(Patient patient) {
+	public Admission getCurrentAdmission(Patient patient) throws OHServiceException {
```

Comment:
The `throws` should not be added as `ioOperations.getCurrentAdmission()` doesn't throw an exception.

Reply:
done, thank you